scripts/experiment_2/nbconverted/4_similarity_analysis_procrustes.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Similarity analysis
+# 
+# We want to determine if the different batch simulated data is able to capture the biological signal that is present in the original data:  How much of the real input data is captured in the simulated batch data?
+# 
+# In other words, we want to ask: “do these datasets have similar patterns”?
+# 
+# To do this we will use a [procrustes analysis](https://en.wikipedia.org/wiki/Procrustes_analysis).
+# 
+# **How does it work?**
+# Given data1 and data2, procrustes manipulates data1 to transform it into data2 through a series of rotations, shifts, scaling in order to minimize the sum of squares of the difference between the two points: $\Sigma(data1 - data2)^2$.
```

Comment:
same comment as hausdorff d. isn't this assuming that each point of the data transformed into another point in the same location of the other data? would that work in this case?

Reply:
I believe that this method is also performing all pairwise comparisons.